docs/redux/redux-intro.md

```diff
@@ -163,119 +167,200 @@ function counter (state = 0, action) {
 			return state;
 	}
 }
-class Counter extends Component {
-	render () {
-		const {value, onIncrement} = this.props;
-		return (
-			<p>
-				Clicked: {value} times
-				{' '}
-				<button onClick={onIncrement}>
-					+
-				</button>
-			</p>
-		);
-	}
+const Counter = ({value, onIncrement}) =>  {
+	return (
+		<p>
+			Clicked: {value} times
+			{' '}
+			<button onClick={onIncrement}>
+				+
+			</button>
+		</p>
+	);
 }
 Counter.propTypes = {
 	onIncrement: PropTypes.func.isRequired,
 	value: PropTypes.number.isRequired
 };
 const store = createStore(counter);
-function render () {
-	ReactDOM.render(
-		<Counter
+const render = () => {
+  const appElement = (
+    <Counter
 			value={store.getState()}
 			onIncrement={() => store.dispatch({type: 'INCREMENT'})}
-		/>,
-		document.getElementById('root')
-	);
+		/>
+  );
+  createRoot(document.getElementById('root')).render(appElement);
 }
 render();
 store.subscribe(render);
 ```
 
-Live Demo: [http://jsbin.com/nemofa/edit?html,js,output](http://jsbin.com/nemofa/edit?html,js,output)
+Live Demo: [https://jsbin.com/pudurig/edit?html,js,output](https://jsbin.com/pudurig/edit?html,js,output)
```

Comment:
Please update the above change to this url.

Reply:
Which one?

Reply:
I meant, I can't see `createRoot` pattern in the Live Demo. https://jsbin.com/pudurig/edit?html,js,output

Reply:
Ah, all right, got it!

Reply:
I'm using stackblitz.com now instead of jsbin because I had some import issues with jsbin while changing the Live Demo sample

Reply:
How about using code sandbox(https://codesandbox.io)?

Reply:
That's even better 👍 I changed it to codesandbox, thanks for the tip!